src/connectors/impls/gcs/resumable_upload_client.rs

```diff
@@ -338,11 +339,15 @@ impl<TClient: HttpClientTrait, TBackoffStrategy: BackoffStrategy + Send + Sync>
 impl<TClient: HttpClientTrait, TBackoffStrategy: BackoffStrategy>
     DefaultClient<TClient, TBackoffStrategy>
 {
-    #[allow(clippy::unnecessary_wraps)] // test requerst a result
```

Comment:
Did you mean? // test requires a result

Reply:
resolving this as it's the delete of the next line but still true :) the message should be better